contracts/AbstractConference.sol

```diff
@@ -156,15 +172,33 @@ contract AbstractConference is Conference, GroupAdmin {
     }
 
     /**
-    * @dev The event owner transfer the outstanding deposits  if there are any unclaimed deposits after cooling period
+    * @dev The event owner transfer the outstanding deposits if there are any unclaimed deposits after cooling period
     */
-    function clear() external onlyAdmin onlyEnded{
-        require(now > endedAt + coolingPeriod, 'still in cooling period');
+    function clear() external onlyAdmin onlyEnded afterCoolingPeriod {        
```

Comment:
As discussed, can you add `require(withdrawn == totalAttended)` so that it can only happen when everybody withdraw?